Lib/os.py

```diff
@@ -1138,7 +1138,17 @@ def add_dll_directory(path):
         )
 
 
-if _exists('sched_getaffinity'):
+def _is_cpu_overrided():
+    env_opt = environ.get('PYTHONCPUCOUNT', None)
+    xopt = sys._xoptions.get('cpu_count', None)
+    if not any((env_opt, xopt)):
+        return False
+    if xopt == 'default':
+        return False
+    return True
```

Comment:
I would prefer to expose PyConfig.cpu_count value using a private function, rather than having two functions parsing env var / cmdline option. Otherwise, the two implementations may become inconsistent later. Also, PyConfig can be modified with the PyConfig API, not only using env var / cmdline.

I'm even working on configuration file-like API: https://github.com/python/cpython/pull/110145